README.md

```diff
@@ -1,41 +1,150 @@
-<a href="https://www.twilio.com">
-  <img src="https://static0.twilio.com/marketing/bundles/marketing/img/logos/wordmark-red.svg" alt="Twilio" width="250" />
+<a  href="https://www.twilio.com">
+<img  src="https://static0.twilio.com/marketing/bundles/marketing/img/logos/wordmark-red.svg"  alt="Twilio"  width="250"  />
 </a>
 
 # Twilio Account Security Quickstart - Twilio Authy and Twilio Verify
 
+![](https://github.com/TwilioDevEd/account-security-quickstart-spring/workflows/Java-Gradle/badge.svg)
+
 > We are currently in the process of updating this sample template. If you are encountering any issues with the sample, please open an issue at [github.com/twilio-labs/code-exchange/issues](https://github.com/twilio-labs/code-exchange/issues) and we'll try to help you.
 
+## About
+
 A simple Java, Spring and AngularJS implementation of a website that uses Twilio Authy to protect all assets within a folder with two-factor authentication. Additionally, it shows a Verify Phone Verification implementation.
 
 It uses four channels for delivery, SMS, Voice, Soft Tokens, and Push Authentication. You should have the [Authy App](https://authy.com/download/) installed to try Soft Token and Push Authentication support.
 
 Learn more about Account Security and when to use the Authy API vs the Verify API in the [Account Security documentation](https://www.twilio.com/docs/verify/authy-vs-verify).
 
+Implementations in other languages:
+
+| .NET | Python | Node | PHP | Ruby |
+| :--- | :--- | :----- | :-- | :--- |
+| TBD | [Done](https://github.com/TwilioDevEd/account-security-quickstart-django)  | [Done](https://github.com/TwilioDevEd/account-security-quickstart-node)  | [Done](https://github.com/TwilioDevEd/account-security-quickstart-php) | [Done](https://github.com/TwilioDevEd/account-security-quickstart-rails)  |
+
+### Features
 
-#### Two-Factor Authentication Demo
 - URL path "/protected" is protected with both user session and Twilio Authy Two-Factor Authentication
 - One Time Passwords (SMS and Voice)
 - SoftTokens
 - Push Notifications (via polling)
-
-#### Phone Verification
 - Phone Verification
-- SMS or Voice Call
+  - SMS or Voice Call
+
+<!--
+### How it works
+
+**TODO: Describe how it works**
+-->
```

Comment:
A bit of intentional. That section is part of the README template but those repos don't have any description of how they work. I think it can be useful to add that later. What do you think?

Reply:
Sure! we can add that description later. I just wanted to make sure it was intentional.